pkg/asset/installconfig/aws/session.go

```diff
@@ -198,12 +202,14 @@ func getUserCredentials() error {
 
 	creds, err := ini.Load(path)
 	if err != nil {
-		if os.IsNotExist(err) {
-			creds = ini.Empty()
-			creds.Section("").Comment = "https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html"
-		} else {
-			return err
+		if !os.IsNotExist(err) {
```

Comment:
How about using os.IsExist(err) here instead of negating os.IsNotExist()?

Reply:
Yeah that's better thanks!

Reply:
Will the error message contain that information? I was under the impression that ini.Load(path) would only return `doesn't exist` and we can only check for that.

Reply:
Looks like you are right. My mistake ;-)